src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -1282,12 +1368,12 @@ private static bool IsTypeSerializable(Type type, HashSet<Type> previousCollecti
         {
             Type itemType;
 
-            if (type.GetTypeInfo().IsEnum ||
+            if (type.IsSerializable ||
+                type.GetTypeInfo().IsEnum ||
```

Comment:
IsEnum is not included here on the full framework. Should this stay or be removed?